src/commands/Help.ts

```diff
@@ -0,0 +1,100 @@
+import { CommandInteraction, Client, MessageEmbed } from 'discord.js';
+import { Command } from '../Command.js';
+import logger from '../utils/logger.js';
+
+const helpEmbed = new MessageEmbed({
```

Comment:
I didn't considered that. But it is ok because the readability in PC was considered, since user will input a lot of characters to this bot which is sticky in the mobile env. And I checked it on mobile, this form is not so weird. Thanks!